cypress/integration/my_patches.js

```diff
@@ -0,0 +1,31 @@
+/// <reference types="Cypress" />
+
+const MY_PATCHES_ROUTE =
+  "/my-patches";
+describe("my patches page", function() {
+  beforeEach(() => {
+    cy.server();
+    cy.login();
+  });
+
+  it("Should render with Show Commit Queue box checked when commitQueue not indicated in URL query param", () => {
+    cy.visit(MY_PATCHES_ROUTE);
+    cy.get(".cy-checkbox")
```

Comment:
some of the old selectors will be classes but we have moved to using data-cy attributes on the elements for testing.
```
// JSX
<div data-cy="commit-queue-checkbox">I'm the checkbox element</div>

// test file
cy.get("[data-cy=commit-queue-checkbox]")
```